app/imports/components/Projects/CreateNewProject/CreateNewProject.jsx

```diff
@@ -35,9 +76,123 @@ class CreateNewProject extends Component {
         }
     }
 
-    renderTextField() {
-        const { isError, isResult } = this.state;
+    addMods(name) {
+        const usersNames = this.state.usersNames.indexOf(name);
+        const chosenModerators = this.state.chosenModerators.indexOf(name);
+
+        if (usersNames !== -1 && chosenModerators === -1) {
+            const chosen = this.state.chosenModerators;
+            chosen.push(name);
+            this.setState({
+                chosenModerators: chosen,
+                inputError: false,
+            });
+        } else {
+            this.setState({
+                inputError: true,
+            });
+        }
+
+        this.modInput.setState({ searchText: '' });
+        this.modInput.focus();
+    }
+
+    addMember(name) {
+        const usersNames = this.state.usersNames.indexOf(name);
+        const chosenMenbers = this.state.chosenMenbers.indexOf(name);
+        const chosenModerators = this.state.chosenModerators.indexOf(name);
+
+        if (usersNames !== -1 && chosenMenbers === -1 && chosenModerators === -1) {
+            const chosen = this.state.chosenMenbers;
+            chosen.push(name);
+            this.setState({
+                chosenMenbers: chosen,
+                memberError: false,
+            });
+        } else {
+            this.setState({
+                memberError: true,
+            });
+        }
+
+        this.memberInput.setState({ searchText: '' });
+        this.memberInput.focus();
+    }
+
+    removeRow(e, index, type) {
+        e.preventDefault();
 
+        if (type === 'mod') {
+            const moderators = this.state.chosenModerators;
+            moderators.splice(index, 1);
+            this.setState({
+                chosenModerators: moderators,
+            });
+        } else {
```

Comment:
If you use `type` with value `mod` you should also use other value for `member`. 
Full name are better, because `mod` can be interpreted as `modifier`.